OWID/OWID_Visualize_GDP_per_capita_through_the_years.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1d69ab6",
+   "metadata": {},
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "267d5bbf",
+   "metadata": {},
+   "source": [
+    "# OWID - Visualize GDP per capita through the years\n",
+    "<a href=\"https://app.naas.ai/user-redirect/naas/downloader?url=https://raw.githubusercontent.com/jupyter-naas/awesome-notebooks/master/OWID/OWID_Visualize_GDP_per_capita_through_the_years.ipynb\" target=\"_parent\"><img src=\"https://naasai-public.s3.eu-west-3.amazonaws.com/Open_in_Naas_Lab.svg\"/></a><br><br><a href=\"https://bit.ly/3JyWIk6\">Give Feedbacks</a> | <a href=\"https://app.naas.ai/user-redirect/naas/downloader?url=https://raw.githubusercontent.com/jupyter-naas/awesome-notebooks/master/Naas/Naas_Start_data_product.ipynb\" target=\"_parent\">Generate Data Product</a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5231f49b",
+   "metadata": {},
+   "source": [
+    "**Tags:** #dash #dashboard #plotly #naas #asset #analytics #dropdown #callback #bootstrap #snippet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b9eabab",
+   "metadata": {},
+   "source": [
+    "**Author:** [Zihui Ouyang](https://www.linkedin.com/in/zihui-ouyang-539626227/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab610df3",
+   "metadata": {},
+   "source": [
+    "**Description:** This notebook creates an interactive plot using Dash app infrastructure with OWID's GDP per capita data. The values are calculated by taking the expenditure-side real GDP at chained PPPs and dividing by the population."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37a304c0",
+   "metadata": {},
+   "source": [
+    "**References:**\n",
+    "- https://www.rug.nl/ggdc/productivity/pwt/?lang=en\n",
+    "- https://stackoverflow.com/questions/70886359/dash-python-making-subplots-when-multiple-parameters-are-selected\n",
+    "- https://dash-example-index.herokuapp.com/line-charts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af4fbd09",
+   "metadata": {},
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0649176e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import dash\n",
+    "    import os\n",
+    "except:\n",
+    "    !pip install dash --user\n",
+    "    import dash\n",
+    "try:\n",
+    "    import dash_bootstrap_components as dbc\n",
+    "except:\n",
+    "    !pip install dash_bootstrap_components --user\n",
+    "    import dash_bootstrap_components as dbc\n",
+    "import pandas as pd\n",
+    "from dash import Dash, html, dcc, callback, Output, Input\n",
+    "import plotly.express as px"
```

Comment:
Import `numpy` before this.

Like `import numpy as np`

Reply:
That is fixed now.